bitnami/spring-cloud-dataflow/templates/skipper/configmap.yaml

```
@@ -96,4 +96,6 @@ data:
         {{- end }}
         testOnBorrow: true
         validationQuery: "SELECT 1"
-{{ end }}
+      flyway:
+        enabled: {{ .Values.flyway.enabled }}
+  {{ end }}
```

The indentation of the `{{ end }}` statement was modified
```suggestion
      flyway:
        enabled: {{ .Values.flyway.enabled }}
{{ end }}
```